extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -28,23 +30,28 @@ class QueueProcessor extends EventEmitter {
      *   consumer group ID
      * @param {String} notifConfig.queueProcessor.retryTimeoutS -
      *   number of seconds before giving up retries of an entry
-     * @param {Object} destinationConfig - destination configuration object
      * @param {String} destinationConfig.type - destination type
      * @param {String} destinationConfig.host - destination host
      * @param {String} destinationConfig.auth - destination auth configuration
      * @param {String} destinationId - resource name/id of destination
      */
-    constructor(mongoConfig, kafkaConfig, notifConfig, destinationConfig,
-        destinationId) {
+    constructor(mongoConfig, kafkaConfig, notifConfig, destinationId) {
         super();
         this.mongoConfig = mongoConfig;
         this.kafkaConfig = kafkaConfig;
         this.notifConfig = notifConfig;
-        this.destinationConfig = destinationConfig;
         this.destinationId = destinationId;
+        this.destinationConfig
+            = notifConfig.destinations.find(dest => dest.resource === destinationId);
+        assert(this.destinationConfig, 'Invalid destination argument. Destination ' +
```

Comment:
It's minor as this is only to debug a config issue, but we could help the admin by logging the `destinationId` with a formatted string